plugins/source_scheduler_dfs.go

```diff
@@ -109,8 +118,9 @@ func (p *SourcePlugin) resolveTableDfs(ctx context.Context, table *schema.Table,
 	}
 
 	// we don't need any waitgroups here because we are waiting for the channel to close
-	if parent == nil { // Log only for root tables, otherwise we spam too much.
-		logger.Info().Uint64("resources", tableMetrics.Resources).Uint64("errors", tableMetrics.Errors).Msg("fetch table finished")
+	if parent == nil { // Log only for root tables and relations only after resolving is done, otherwise we spam per object instead of per table.
+		logger.Info().Uint64("resources", tableMetrics.Resources).Uint64("errors", tableMetrics.Errors).Msg("table sync finished")
+		p.logTablesMetrics(table.Relations, client)
```

Comment:
```suggestion
		p.logTablesMetrics(table.Relations, client)
```

[non blocking] Maybe this could be `p.logTableMetrics(table, client)` and move the log in the above line into the method? This way we don't need to duplicate it